### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by alexjoverm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,70 @@
...
 
-BindGroup supports any control field that uses `onChange` to deal with changes and it does by default, but additionally you can provide other event reporter with `data-event` as follows:
+### Get started
 
-```javascript
-<input data-bind="value0" data-event="onInput"/>
+Install it:
+
+```bash
+npm install --save-dev preact-bind-group
```
Comment (k1r0s) [inline README.md]: @alexjoverm why `--save-dev`?

Comment (alexjoverm) [reply README.md]: Good question, not needed :)

File: README.md [hunk 2]
```diff
@@ -1,71 +1,70 @@
...
-      <input data-bind="value2"/>
-    </div>
-    <div>
-      <input data-bind="value3"/>
-    </div>
-  </BindGroup>
-</div>
-```
-If you change any input value you should see this on the console:
+The `watch` callback receives an `{key: value}` object containing the changed property as its parameter.
```
Comment (k1r0s) [inline README.md:48]: I would like cbk signature to be here  `({ [key: string]: any }, key: string) => void`.

Comment (alexjoverm) [reply README.md:48]: Ok

**Pull request merged**